react/hooks/useEventListener.spec.js

```diff
@@ -30,6 +47,15 @@ describe('useEventListener', () => {
     expect(result.error).not.toBeDefined()
   })
 
+  it('should not subscribe for an undefined ref element', async () => {
+    const cb = jest.fn()
+    const ref = {
+      current: undefined
+    }
+    const { result } = renderHook(() => useEventListener(ref, 'click', cb))
+    expect(result.error).not.toBeDefined()
```

Comment:
Why should result.error be not defined ?

Reply:
I done as [line 47](https://github.com/cozy/cozy-ui/pull/2417/files/800fda4f3b1081c7012afddcb7bdb2268b35916d#diff-05864bb9fd827c4bc4b3a26e900bb6e37d706e8213eee55251fbfee49bd53e9eR47) it would have an error if we tried to add an event listened on nothing